NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/TransferGroup.java

```diff
@@ -52,12 +52,6 @@ public boolean execute(CommandSender sender, String[] args) {
 			return true;
 		}
 		
-		if (!gm.hasAccess(g, uuid, PermissionType.getPermission("TRANSFER")) && 
```

Comment:
I can't really think of a compelling reason it shouldn't be; sure you could "delegate" the work, but you might as well just promote that person to owner then... so it all winds up back at primary owner.